rollup.config.mjs

```diff
@@ -0,0 +1,64 @@
+import {readFileSync} from 'fs';
```

Comment:
Nitpicking: We can remove this line now.

Reply:
Good catch! Fixed.